src/examples/src/widgets/breadcrumb/Basic.tsx

```diff
@@ -0,0 +1,10 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Breadcrumb from '@dojo/widgets/breadcrumb';
+
+const factory = create();
+
+const App = factory(function() {
+	return <Breadcrumb current="step" item={{ label: 'Step 1' }} />;
```

Comment:
can the item properties be changed to `BreadcrumbProperties` ?
ie. `<Breadcrumb current='step' label='step1' />` etc?
When rendering them you can spread the item over the widget. ie. `<Breadcrumb current='step' {...item} />`